doc/classes/@GlobalScope.xml

```diff
@@ -1196,6 +1196,29 @@
 				else:
 				    print("Unexpected result")
 				[/codeblock]
+				See also [method typeof_string]
+			</description>
+		</method>
+		<method name="typeof_string">
+			<return type="String" />
+			<param index="0" name="type" type="int" />
+			<description>
+				Returns a human-readable name of the given [param type], using the [enum Variant.Type] values.
+				[codeblock]
+				var values = [1, 2.5, "Hello", true]
+				
+				for value in values:
+					var type_id = typeof(value)
+					var type_name = typeof_string(type_id)
+					print("%s is %s" % [value, type_name])
```

Comment:
You don't need string formatting or looping through an array to explain this concept.